stripe/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -114,23 +138,74 @@ class PaymentSheet internal constructor(
         }
     }
 
+    /**
+     * A class that presents the individual steps of a payment sheet flow.
+     */
     internal interface FlowController {
-        fun getPaymentOption(): PaymentOption?
 
-        fun configure(
+        /**
+         * Configure the FlowController to process a [PaymentIntent].
+         *
+         * @param[paymentIntentClientSecret] the client secret for the [PaymentIntent].
+         * @param[configuration] optional [PaymentSheet] settings.
+         * @param[callback] called with the result of configuring the FlowController.
+         */
+        fun configureWithPaymentIntent(
             paymentIntentClientSecret: String,
-            configuration: Configuration,
+            configuration: Configuration? = null,
```

Comment:
Can we leave this as non-null?

Reply:
The default parameters are needed for the @JvmOverloads to create the overloaded methods for Java.

Reply:
But this is for the interface, so we can't use @JvmOverloads?

Reply:
Okay, I though this comment was on `PaymentSheet`.